src/utils/provider.ts

```diff
@@ -26,6 +26,12 @@ import type { GetTransactionReceiptResponse } from './transactionReceipt';
 
 /**
  * Helper - Async Sleep for 'delay' time
+ *
+ * @param {number} delay - Number of milliseconds to delay.
+ * @example
+ * ```typescript
+ * wait(1000) // 1000 milliseconds == 1 second
```

Comment:
It's an async function.
You have to add the `await` prefix.
All the functions of this file are in the `provider` space. To correct.